app/views/rooms/show.html.erb

```diff
@@ -0,0 +1,47 @@
+<p id="notice"><%= notice %></p>
+
+<p>
+  <strong>Floor:</strong>
+  <%= @room.floor %>
+</p>
+
+<p>
+  <strong>Room number:</strong>
+  <%= @room.number %>
+</p>
+
+<p>
+  <strong>Room name:</strong>
```

Comment:
Man könnte Floor und Room Number eigentlich auch weglassen, weil die Informationen schon in Room Name enthalten ist, oder? Auf jeden Fall würde ich Room Name an den Anfang der Seite setzen und nicht erst an dritte Stelle.

Reply:
Bei `Room Number` und `Room Name` könnte man Room auch weglassen und nur `Number` und `Name` schreiben.

Reply:
Nein, Room Name enthält nicht unbedingt Number und Floor.